collect_app/src/androidTest/java/org/odk/collect/android/support/pages/NotificationDrawer.kt

```diff
@@ -79,6 +79,16 @@ class NotificationDrawer {
     ): D {
         val device = waitForNotification(appName, title)
         device.findObject(By.text(title)).click()
+
+        /*
+        It appears that sometimes the notification drawer does not close automatically
+        after clicking on a notification. This could be due to a bug in Android.
+         */
+        val manageButton = device.findObject(By.text("Manage"))
```

Comment:
I like this solution. My only worry here is that we might end up with a timing issue if "Manage" hangs around for a small amount of time. Maybe we can add a wait in here (like we do in `waitForNotification`) that waits for it to disappear (and times out).

I'm also thinking we should pull this out to a method and do it everywhere we're expecting the notification drawer to close?

Reply:
> My only worry here is that we might end up with a timing issue if "Manage" hangs around for a small amount of time. Maybe we can add a wait in here (like we do in waitForNotification) that waits for it to disappear (and times out).

I've added waiting for 1s but it shouldn't be necessary as we perform other actions before like clicking on a notification or its action so that button should be already there at the time we try to find it.

> I'm also thinking we should pull this out to a method and do it everywhere we're expecting the notification drawer to close?

Yeah, we can do that.